contracts/SwapKiwiV1.sol

```diff
@@ -0,0 +1,286 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.1;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract SwapKiwiV1 is Ownable, IERC721Receiver {
```

Comment:
Why do we still need SwapKiwiV1 contract?

Reply:
It's not deployed, we left it here for reference as it's still in production and it would be hard to find for non-developers if we just delete it.